Add disclaimer that the Wii's IOSses should not be confused with Apple's iOS (#74)
hacks-guide/Guide_Wii: _pages/en_US/cios.md

* An SD card or USB drive
* [d2x cIOS Installer](/assets/files/d2x-cios-installer.zip)

> It is helpful to note that IOS on the Wii is not affiliated with iOS on Apple devices, and does not bare much technical similarities internally. Do not get the Wii IOSs and Apple's iOS mixed up, as it may cause confusion.

Reviewer: One typo, two style suggestion:
```suggestion
It is helpful to note that IOS on the Wii is not related to iOS on Apple devices, and does not bear much technical similarities internally. Do not get the Wii IOSs and Apple's iOS mixed up.
```
